docs/dev/vagrant.rst

```diff
@@ -105,26 +94,19 @@ Once you have everything set up, normal usage of your vagrant dev server should
 Before you start anything: ::
 
     vagrant up
-    fab open_db
 
 To run your dev server: ::
 
-    fab run_devserver
+    fab runserver
 
 Other useful command examples: ::
 
     fab manage:shell_plus
-    fab manage:'migrate program'
```

Comment:
The point of having this in the examples was to demonstrate use of quotes. I'm inclined to leave it in, unless you can think of something more useful to replace it with.